src/Network/Request.php

```diff
@@ -134,7 +134,7 @@ class Request implements ArrayAccess
         'json' => ['accept' => ['application/json'], 'param' => '_ext', 'value' => 'json'],
         'xml' => ['accept' => ['application/xml', 'text/xml'], 'param' => '_ext', 'value' => 'xml'],
     ];
-
+    
```

Comment:
You're right, damned Netbeans :-(

I apologize if I have to do all these PR, slowly I will become smarter.